### PR TITLE
WIP: Testing stats

### PR DESCRIPTION
**Pull request by larsoner**
What I'm currently using for testing.

### CODE REVIEW

File: scripts/doc/Makefile [hunk 1]
```diff
@@ -238,17 +241,15 @@ install:
...
-	touch _build/mne-biomag-group-demo.github.io/.nojekyll
-	cd _build/ && \
-	cp -r html/* mne-biomag-group-demo.github.io && \
-	cd mne-biomag-group-demo.github.io && \
-	git add * && \
-	git add .nojekyll && \
-	git commit -a -m 'Make install' && \
-	git push
+	git clone -b gh-pages --single-branch --no-checkout --depth 1 git@github.com:/$(USER)/mne-biomag-group-demo _build/mne-biomag-group-demo.github.io
+	@make install-files
```
Comment (agramfort) [inline scripts/doc/Makefile:252]: what is the diff between install-pr-user-ssh and install-pr-user ?

it seems to be the same

Comment (larsoner) [reply scripts/doc/Makefile:252]: you're right, I need to revert to using `https://` in this one

**Comment by jasmainak** [comment]
@larsoner did we need to merge this PR?

**Comment by larsoner** [comment]
No

**Pull request closed without merging**